sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
-regression, or ranking)
-- Introduce ligand-based approaches, hype and excitement surrounding
-performance of a "high-parameter" network on the Merck Kaggle challenge,
-cover other neural networks trained on fingerprints or descriptors as features
-that followed, Tox21 Data Challenge
-- Multitask networks related to the above point
-- Realistic view of where things stand today, high-parameter networks struggle
-with overfitting, cross validation needs to be done carefully because of temporal
-structure [@tag:Kearnes2016_admet], low parameter networks based on chemical
-similarity (IRV) work very well, especially well-suited for the domain in which
-training data can be limited and contains few positive instances, may touch on
-BACE example here and other discussions of training data limitations (e.g.
-[@tag:AltaeTran2016_one_shot])
-- "Creative experimentation" phase of the field, new ideas for representation
-learning and novel approaches including graph convolutions, autoencoders,
-one shot learning, and generative models
-- These "creative" approaches are definitely interesting but aren't necessarily
-outperforming existing methods, improvements on the software and
-reusability side could be important to help establish more rigorous
-benchmarking, DeepChem as example of this
-- Future outlook, what would need to happen for the "creative" approaches
-to overtake the current state of the art, can representation learning be
-improved by incorporating more information about chemical properties or
-even more "tasks" during training, how much will future growth depend on
-data versus algorithms
-- Future outlook part 2, how the above approaches relate to traditional
-methods like docking (note neural networks that include docking scores as
-features), deep learning efforts in this direction that use structure (e.g.
-[@tag:Wallach2015_atom_net @arxiv:1612.02751]), "zero-shot learning",
-analogies to other domains where deep learning can capture the behavior
-of complex physics (e.g. quantum physics example), maybe briefly mention
-other compound-protein interaction-based networks although that doesn't seem
-to fit here and is somewhat out of scope
-- Future output part 3 (most speculative), what would successful generative
-networks mean for the HTS field?
+In the biomedical domain, high-throughput chemical screening aims to improve
+therapeutic options over a long term horizon [@tag:PerezSianes2016_screening]
+`TODO: add another general screening reference`.  The objective is to discover
+which small molecules (also referred to as chemical compounds or ligands)
+effectively and specifically affect the activity of a target, such as a kinase,
+protein-protein interaction, or broader cellular phenotype.  This screening
+process can serve as the first step in the long drug discovery pipeline, where
+novel chemicals are pursued for their ability to inhibit or enhance
+disease-relevant biological mechanisms.  The appeal of machine learning in this
```

Comment:
Do you need multiple hits though, or is one hit enough? Is the goal to identify a bunch of hits and then follow up with actual screens on only those hits? Does this help to give the most lead compound options in case one has to be abandoned due to off target activity?